imports/ui/QuestionSidebar.jsx

```diff
@@ -29,13 +29,15 @@ export class QuestionSidebar extends ControlledForm {
     super(props)
```

Comment:
When I click to create a new question, I first get a client side error in the JS console about the question prop being required for QuestionEditItem, but the question is undefined. I think the question editor should create an empty question if it is not passed a prop; perhaps that is simply fixed by not making question a required prop.


This is then followed by : Exception in delivering result of invoking 'questions.insert': TypeError: Cannot read property '_id' of undefined.

Presumably, you have to make sure that the user has made a change to the blank question before trying to save it the first time.